_inc/client/general-settings/site-plan.jsx

```diff
@@ -48,7 +48,6 @@ export const SitePlan = React.createClass( {
 export default connect(
 	( state ) => {
 		return {
-			getSiteConnectionStatus: () => getSiteConnectionStatus( state ),
```

Comment:
Why are we not using the connected action selector?  Won't this result in this not updating along with the state?